### ISSUE TITLE
Cuff and uncuff!?!

### ISSUE THREAD

**Issue by Armagryzor** [Sighmir/vrp_hotkeys#2]
Hi Sighmir,

On the first accept my excuse because i know that is not an issue.

Please , can you add on your script cuff /uncuff and open/close vehicle.

Thank you very much!!!!

**Comment by Sighmir** [comment]
Yes you already asked for it multiple times in other issues and I said its on my FOREVER GROWING to do list of over 30 items. I am just one person ask only once and wait. And as I said this is an interface for you to make your own hotkeys, don't expect me to add every single feature to it. 
I will add those as soon as I have chance but please understand, I am not being paid to make scripts except for my donators which I am extremely thankful for. I have a half time job and am going through university, I wake up at 5am leave at 7am and only get back home for lunch before work and then only back again at 10pm, I only make scripts on Fridays and weekends except for quick fixes I can push out from work or at lunch.
Please be patient.

**Comment by Sighmir** [comment]
Duplicate of #1

**Comment by Sighmir** [comment]
Which key would you like cuff and uncuff to go on ? I am not sure where I am gonna put it yet.

**Comment by Armagryzor** [comment]
Thank you Sighmir ..

Is is possible leftshift+ Z ?

Thank you

**Comment by Sighmir** [comment]
Hum haven't made a hotkey with 2 keys yet but I could try. I dont really think it will be a problem!

**Comment by Sighmir** [comment]
I think now both options are added.
